pallets/capacity/src/lib.rs

```diff
@@ -158,6 +164,21 @@ pub mod pallet {
 	pub type CapacityLedger<T: Config> =
 		StorageMap<_, Twox64Concat, MessageSourceId, CapacityDetails<BalanceOf<T>, T::BlockNumber>>;
 
+	/// Storage for the current epoch number
+	#[pallet::storage]
+	#[pallet::getter(fn get_current_epoch)]
+	pub type CurrentEpoch<T: Config> = StorageValue<_, T::BlockNumber, ValueQuery>;
```

Comment:
Let's avoid using  T::BlockNumber here and create our epoch type. We should not be able to add an epoch with a block number. Not a blocker for this PR.